charts/travis-build/values.yaml

```diff
@@ -0,0 +1,48 @@
+# Default values for travis-build.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: travisci/travis-build
+  tag: aj-docker
```

Comment:
should we change this to `latest`?

Reply:
I think in general we should try to version and tag the Docker images we deploy. Using `latest` makes it harder to deploy new code, because nothing inside the pod template changes, so Kubernetes won't actually do anything to deploy new code.

I agree that we should change the tag here before merging this. Our other projects are using the `vX.Y.Z-<blah>` scheme for tags that we get from `git describe`. I'd like to continue using that same pattern unless we have a particular reason to do something else.

Reply:
I can set up an automated build on Docker Hub if that works. What goes in the `<blah>` part?

Reply:
looks like we're pushing images of this app to Quay, per https://github.com/travis-pro/dev/issues/43 , gonna take a look to see if we can push them to Docker Hub instead (or porque no los dos?)

Reply:
I don't think it matters too much which registry we upload to. @meatballhat had suggested to me that we were using Quay less these days, but Kubernetes can use it if we want. There is a bit of extra complexity in using private Docker images, but not too much AFAIK. That said, I don't see why travis-build couldn't be public.

The other infra projects have been pushing Docker images in their Travis CI builds. The command used to generate the version string for the image tag is `git describe --always --dirty 2> /dev/null`. That gives you something like `v1.0.0-3-ae37982`: `<most recent tag>-<commits since the tag>-<commit sha>`.

Reply:
So, after looking into this a bit further, I think our no.1 problem is that we don't actually do any sort of versioning of travis-build. I think we might want to address that before we continue with any sort of version pinning for Enterprise, but that requires a workflow change for other teams.

As per https://hub.docker.com/r/travisci/travis-build/tags/ , we are already building new Docker images on Hub every time that we update `master` -- the problem here is that we aren't building new docker images on tags, because we're not tagging those versions.

We could, in the meantime, version by commit, if we wanted to, or create tags manually when we want to create a new Enterprise release...? 🤔

Reply:
I think versioning by commit would be better than just updating latest, so that would be a start. IMO it would serve us well to be able to track versions for the sake of understanding what we are deploying in Enterprise (and it may be nice in hosted eventually too), but it isn't exactly blocking.